<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3d Viewer</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string>color:black;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>961</width>
      <height>450</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QScrollBar::handle:vertical {	
	background-color: rgb(80, 80, 122);
	min-height: 30px;
	border-radius: 7px;
}

background: white;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="group_Move">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>470</y>
      <width>300</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black</string>
    </property>
    <property name="title">
     <string>Перемещение</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_MoveX">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_MoveY">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>60</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_MoveZ">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_Move">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>260</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white; background-color:  rgb(0, 43, 200);  border-radius: 10px</string>
     </property>
     <property name="text">
      <string>Переместить</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_MoveX">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>17</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>dx:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_MoveY">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>17</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>dy:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_MoveZ">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>17</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>dz:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_Move_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>470</y>
      <width>300</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black</string>
    </property>
    <property name="title">
     <string>Масштабирование</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_ScaleX">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black;</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ScaleY">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>60</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ScaleZ">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_Scale">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>260</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white; background-color:  rgb(0, 43, 200);  border-radius: 10px</string>
     </property>
     <property name="text">
      <string>Масштабировать</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ScaleX">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>15</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>kx:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ScaleY">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>15</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ky:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ScaleZ">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>15</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>kz:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="group_Move_3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>470</y>
      <width>300</width>
      <height>120</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black</string>
    </property>
    <property name="title">
     <string>Поворот</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="lineEdit_TurnX">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_TurnY">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>60</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_TurnZ">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">color: black</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="button_Turn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>260</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white; background-color:  rgb(0, 43, 200);  border-radius: 10px</string>
     </property>
     <property name="text">
      <string>Повернуть</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_TurnX">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>17</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>ox:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_TurnY">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>17</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>oy:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_TurnZ">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>17</width>
       <height>18</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>oz:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1030</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOOP_Lab1_Obergan">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuOOP_Lab1_Obergan"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
